app.js

```diff
@@ -1,19 +1,28 @@
 const db = require("./db/connection");
 const express = require("express");
-const { getCategories } = require("./controllers/games.controller");
+const { getCategories, getReviewById } = require("./controllers/games.controller");
 
 const app = express();
 
 app.use(express.json());
 
 app.get("/api/categories", getCategories);
 
+app.get("/api/reviews/:review_id", getReviewById);
+
 app.use('*', (req, res, next) => {
     next(err)
```

Comment:
instead of invoking next here you should be doing your res.status and res.send from here with the invalid path error message